test/e2e/specs/fakerp/carotation.go

```diff
@@ -28,14 +28,18 @@ var _ = Describe("Ca Rotation E2E tests [CaRotation][Fake][LongRunning]", func()
 		Expect(after).NotTo(BeNil())
 
 		By("Verifying that the correct ca certificates have been updated")
+		Expect(before.Config.Certificates.Ca.Cert).To(Equal(after.Config.Certificates.Ca.Cert))
```

Comment:
Do we also want to rotate this CA, or is this a CA we can't rotate?

Reply:
doing them bit by bit. This is a harder one as it's used all over the place.